brianlong/www.validators.app
# Bump rails to version 6.1.7

#### What's this PR do?
- Migrates Rails to version 6.1.7

#### How should this be manually tested?
- run tests
- run an app on staging and click through UI, confirm that no script crashes etc.

#### What are the relevant tickets?
- [Tracker 182802882](https://www.pivotaltracker.com/story/show/182802882)

#### Task completed checklist:
- [N/A] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [N/A] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
